src/main/webapp/WEB-INF/tags/student/home/courseTable.tag

```diff
@@ -3,13 +3,15 @@
 <%@ taglib tagdir="/WEB-INF/tags/student/home" prefix="home" %>
 <%@ tag import="teammates.common.util.Const" %>
 <%@ attribute name="sessionRows" type="java.util.Collection" required="true" %>
-<table class="table-responsive table table-striped">
+<table class="table-responsive table table-striped table-bordered margin-0">
```

Comment:
Can explain this change?

Reply:
Oops, just saw your response above https://github.com/TEAMMATES/teammates/pull/8677#discussion_r178464651

Reply:
I feel it's outside the scope of this PR though. You can open a new issue if you'd like to add the vertical lines.

Reply:
I added vertical lines in this PR because it was necessary to differentiate the columns after adding sort functionality in StudentHomePage. Without this, it is confusing for user to know that sort glyphicon belongs to deadline column, and not the one beside it.

Reply:
Oh okay; it's fine then :+1: